From: Qball Cow <qball@sarine.nl>
Date: Mon, 1 Nov 2010 22:57:27 +0100
Subject: Fix building with vala 0.11 something

---
 src/vapi/libmpd.vapi |    2 --
 1 files changed, 0 insertions(+), 2 deletions(-)

diff --git a/src/vapi/libmpd.vapi b/src/vapi/libmpd.vapi
index 9a5c2c5..f3a84a6 100644
--- a/src/vapi/libmpd.vapi
+++ b/src/vapi/libmpd.vapi
@@ -46,8 +46,6 @@ namespace MPD {
         public unowned MPD.Song copy ();
         [CCode (instance_pos = -1)]
         public void markup (char[] buffer, int length, string markup);
-        [CCode (cname = "mpd_freeSong")]
-        public static void free;
     }
     namespace Sticker {
         namespace Song {
-- 
